sdk/documenttranslation/azure-ai-documenttranslation/samples/async_samples/sample_batch_translation_async.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os
+import asyncio
+
+
+class BatchTranslationSampleAsync(object):
+
+    async def batch_translation_async():
+        # import libraries
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.textanalytics.aio import DocumentTranslationClient
+        from azure.ai.documenttranslation import (
+            BatchDocumentInput,
+            StorageTarget
+        )
+
+        # get service secrets
+        endpoint = os.environ["AZURE_DOCUMENT_TRANSLATION_ENDPOINT"]
+        key = os.environ["AZURE_DOCUMENT_TRANSLATION_KEY"]
+        source_container_url = os.environ["AZURE_SOURCE_CONTAINER_URL"]
+        target_container_url_es = os.environ["AZURE_TARGET_CONTAINER_URL_ES"]
+        target_container_url_fr = os.environ["AZURE_TARGET_CONTAINER_URL_FR"]
+
+        # create service client
+        client = DocumentTranslationClient(endpoint, AzureKeyCredential(key))
+
+        # prepare translation job input
+        batch = [
+            BatchDocumentInput(
+                source_url=source_container_url,
+                source_language="en",
+                targets=[
+                    StorageTarget(
+                        target_url=target_container_url_es,
+                        language="es"
+                    ),
+                    StorageTarget(
+                        target_url=target_container_url_fr,
+                        language="fr"
+                    )
+                ],
+                storage_type="file"
+            )
+        ]
+
+        # run translation job
+        async with client:
+            job_detail = await client.create_translation_job(batch)
+
+            print("Job initial status: {}".format(job_detail.status))
+            print("Number of translations on documents: {}".format(job_detail.documents_total_count))
+
+            # get job result
+            job_result = await client.wait_until_done(job_detail.id)  # type: JobStatusDetail
+            if job_result.status == "Succeeded":
+                print("We translated our documents!")
+                if job_result.documents_failed_count > 0:
+                    check_documents(client, job_result.id)
```

Comment:
For this call and below on line 68

```suggestion
                    await self.check_documents(client, job_result.id)
```